DQM/L1TMonitor/python/L1TdeStage2uGMT_cff.py

```diff
@@ -92,43 +92,43 @@
 # compares the unpacked uGMT intermediate muon collection to the emulated uGMT intermediate muon collection
 # only muons that do not match are filled in the histograms
 l1tdeStage2uGMTIntermediateBMTF = l1tdeStage2uGMT.clone()
-l1tdeStage2uGMTIntermediateBMTF.muonCollection1 = cms.InputTag(unpackerModule, "imdMuonsBMTF")
-l1tdeStage2uGMTIntermediateBMTF.muonCollection2 = cms.InputTag(emulatorModule, "imdMuonsBMTF")
-l1tdeStage2uGMTIntermediateBMTF.monitorDir = cms.untracked.string(ugmtEmuImdMuDqmDir+"/BMTF/data_vs_emulator_comparison")
-l1tdeStage2uGMTIntermediateBMTF.summaryTitle = cms.untracked.string("Summary of uGMT intermediate muon from BMTF comparison between unpacked and emulated")
-l1tdeStage2uGMTIntermediateBMTF.ignoreBin = cms.untracked.vint32(ignoreBins)
+muonCollection1 = (unpackerModule, "imdMuonsBMTF")
+muonCollection2 = (emulatorModule, "imdMuonsBMTF")
+monitorDir = ugmtEmuImdMuDqmDir+"/BMTF/data_vs_emulator_comparison"
+summaryTitle = "Summary of uGMT intermediate muon from BMTF comparison between unpacked and emulated"
+ignoreBin = cms.untracked.vint32(ignoreBins)
```

Comment:
can this type be dropped?

Reply:
I think it can't be dropped as it is not already defined. While I tried to remove type it was showing error.

Reply:
Yes. You can define a default value in `l1tdeStage2uGMT` (empty vector), so that it can be modified everywhere else; or alternatively let the type specified, as it is now (not to be preferred, though)